jsapp/js/account/plans/planButton.component.tsx

```diff
@@ -0,0 +1,99 @@
+import BillingButton from 'js/account/plans/billingButton.component';
+import React, {useCallback} from 'react';
+import styles from 'js/account/plans/plan.module.scss';
+import {
+  BasePrice,
+  Organization,
+  Price,
+  SubscriptionInfo,
+} from 'js/account/stripe.types';
+import {postCustomerPortal} from 'js/account/stripe.api';
+import {
+  getSubscriptionsForProductId,
+  processCheckoutResponse,
+} from 'js/account/stripe.utils';
+
+interface PlanButtonProps {
+  price: Price;
+  isBusy: boolean;
+  plans: SubscriptionInfo[] | null;
+  setIsBusy: (value: boolean) => void;
+  hasManageableStatus: (subscription: SubscriptionInfo) => boolean;
+  isSubscribedProduct: (product: Price) => boolean;
+  buySubscription: (price: BasePrice) => void;
```

Comment:
Could we follow the `on…` naming convention here please in all the functions? :) Here we could have `onBuySubscriptionRequested`. I know it's a mouthful, but it's more clear and precise